composer.json

```diff
@@ -7,15 +7,20 @@
     "license": "Apache-2.0",
     "require": {
         "php": ">=5.4",
-        "google/auth": "dev-master"
+        "google/auth": "dev-master",
+        "monolog/monolog": "^1.17",
+        "phpseclib/phpseclib": "~2.0"
     },
     "require-dev": {
         "phpunit/phpunit": "3.7.*",
         "squizlabs/php_codesniffer": "~2.3"
     },
     "autoload": {
+        "psr-0": {
+            "Google_": "src/"
+        },
         "classmap": [
-            "src/"
+            "src/Google/Service/"
         ]
```

Comment:
Hey Matt! Thanks for the heads up. Yeah, I've noticed the newer docs are for 6. This was on my list but a lower priority. We will definitely look into the effort required to upgrade.